### PR TITLE
Add panel support for wt865x8

### PR DESCRIPTION
**Pull request by jenneron**

### CODE REVIEW

**Comment by Minecrell** [comment]
Please drop your `drm/panel: add panel driver for Wingtech Tianma HX8394D` commit, add the panel driver to https://github.com/msm8916-mainline/linux-panel-drivers and run the `generate.sh` script to automatically generate a `squash!` commit that adds the panel driver. I guess I didn't mention this anywhere yet, sorry about that :)

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi [hunk 1]
```diff
@@ -29,10 +39,70 @@
 	};
 };
 
+&blsp_i2c2 {
+	status = "okay";
+
+	regulator@3e {
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi:45]: Nothing to change here, just want to say :+1: for changing this from `tps65132@3e` -> `regulator@3e`

File: arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi [hunk 2]
```diff
@@ -29,10 +39,70 @@
...
+		compatible = "ti,tps65132";
+		reg = <0x3e>;
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&tps65132_en_default>;
+
+		reg_lcd_pos: outp {
+			regulator-name = "outp";
+			enable-gpios = <&msmgpio 118 GPIO_ACTIVE_HIGH>;
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi:54]: `regulator-active-discharge = <1>` is gone?

Comment (jenneron) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi:54]: I thought we decided to remove it due to missing discharge gpio

Comment (Minecrell) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi:54]: Oh no, what I meant is that `ti,active-discharge-time-us` works only with the discharge gpio. The discharge stuff itself should work fine.

Comment (jenneron) [reply arch/arm64/boot/dts/qcom/msm8916-wingtech-wt865x8.dtsi:54]: Okay, i will return it.

**Review by Minecrell — APPROVED** [review]

**Pull request merged**